pkg/database/database.go

```diff
@@ -90,3 +89,18 @@ func NewDatabase(ctx context.Context, log *logrus.Entry, env env.Interface, m me
 
 	return db, nil
 }
+
+func NewJSONHandle(cipher encryption.Cipher) *codec.JsonHandle {
```

Comment:
NewJSONHandle shouldn't be exported.  I don't think it's legit to use it from other packages.

Reply:
This way we can make sure we configure it in one and single place. If we want to keep it in the `NewDatabase` method we should refactor trigger creation in the individual clients. I think the price to avoid sensitive code duplication is ok here? otherwise all other places we configure json handler we copy-paste?